lib/common/tests/xml/pcmk__xml_init_test.c

```diff
@@ -19,6 +19,6 @@ schemas_initialized(void **state)
     assert_non_null(pcmk__find_x_0_schema());
 }
 
-// The group setup/teardown functions call pcmk__xml_init()/pcmk__xml_cleanup()
+// The group setup/teardown functions call pcmk__xml_init() do cleanup
```

Comment:
Typo here - I think this is supposed to be `to cleanup`.

Reply:
"and do cleanup" -- thanks!

Reply:
This whole unit test is supposed to go away though. That's the real oversight lol.